cypress/integration/task_actions.ts

```diff
@@ -0,0 +1,45 @@
+/// <reference types="Cypress" />
+
+describe("Task Action Buttons", function() {
+  before(() => {
+    cy.login();
+  });
+  beforeEach(() => {
+    cy.preserveCookies();
+  });
+
+  it("Some buttons should be disabled and others should be clickable. Clicking on buttons produces banners.", () => {
```

Comment:
this nor the tests tell me how this feature works. why are some buttons disabled and why are others clickable? why does clicking on buttons produce banners?

Reply:
I just updated the tests